packages/styles/src/tokens/_device.scss

```diff
@@ -0,0 +1,19 @@
+@use '@swisspost/design-system-tokens/device' as device;
+
+@use './core';
+@use './../mixins/tokens';
+@use './../mixins/media';
+
+:root {
+  @include media.max(600px) {
+    @include tokens.from(device.$post-mobile);
+  }
```

Comment:
These are the default values and may not be included in a media query. Are you sure this works?

Reply:
By being specific, there's reduced complexity in the devtools because the default tokens don't have to be overwritten. The way the queries are set up, max, between, min, every possible screen size has been covered.